docs/application/native/api/iot-headed/5.5/group__Evas__Object__Group.html

```diff
@@ -2632,7 +2660,7 @@
 <div class="memdoc">
 
 <p>Get the "static clipper" hint flag for a given Evas object. </p>
-<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> if it's to be used as a static clipper, <code>false</code> otherwise. </dd></dl>
+<dl class="return"><dt><b>Returns:</b></dt><dd><code>true</code> if it's to be used as a static clipper, <code>false</code> otherwise.  2.3 </dd></dl>
```

Comment:
Would you please confirm this update?

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/